trunk/Lib/DynamicConditions.php

```diff
@@ -148,7 +148,7 @@ private function defineAdminHooks() {
         $this->loader->addAction( 'elementor/element/popup/section_advanced/after_section_end', $pluginAdmin, 'addConditionFields', 10, 3 );
 
         $this->loader->addAction( 'admin_notices', $pluginAdmin, 'addAdminNotices', 10, 3 );
-        $this->loader->addAction( 'admin_enqueue_scripts', $pluginAdmin, 'enqueueStyles' );
+        #$this->loader->addAction( 'admin_enqueue_scripts', $pluginAdmin, 'enqueueStyles' );
         $this->loader->addAction( 'elementor/editor/before_enqueue_styles', $pluginAdmin, 'enqueueStyles' );
```

Comment:
This line results in the plugin trying to load the same css file that was just removed. Comment it out or adjust.
trunk/Admin/DynamicConditionsAdmin.php and remove the css their. 
the last would be prefarable in my eyes.

Reply:
Sometimes i´m blind :-) Good that we do code-reviews.
It is commented out now.